examples/basic/lines/line_single.py

```diff
@@ -1,3 +1,14 @@
+''' A line graph using user-defined data points. This example
+demonstrates the use of the ``line`` function to make a graph
+by mapping straight linesbetween defined points.
```

Comment:
minor edits:
```suggestion
demonstrates the use of the ``line`` method to make a graph
by drawing straight lines between defined points.
```

Reply:
I'm going to go ahead and commit the suggestion above so this can be  merged today. Thanks for the PR @Christophers27 !